src/components/modern/AssetListItem/AssetListItem.js

```diff
@@ -64,10 +66,11 @@ function AssetListItem({
       <Icon url={iconUrl} />
       <Name>{name}</Name>
 
-      {!!symbol && balance != null && (
+      {!!symbol && !!address && balance != null && (
         <Balance
           chain={chain}
-          symbol={symbol}
+          assetSymbol={symbol}
```

Comment:
Similar to other componens, consider dropping assetSymbol to simplify component usage.

Reply:
Per our internal discussion added this as to do: get assetSymbol from matching asset once assets can be queried by assetAddress as key instead of performing expensive search on whole assets array.